test/integration/override_package/serverspec/default_spec.rb

```diff
@@ -6,19 +6,19 @@
   it_behaves_like 'elasticsearch user',     user: 'elasticsearch',
                                             uid: 1111,
                                             shell: '/bin/sh',
-                                            group: 'bar',
+                                            group: 'elasticsearch',
```

Comment:
Same question here as above. Many folks were overriding these values and adding these tests a while back helped prevent a whole class of bug reports.